src/Form/FormAnalyzer.js

```diff
@@ -262,7 +274,8 @@ class FormAnalyzer {
         if (
             (el instanceof HTMLAnchorElement && el.href && el.getAttribute('href') !== '#') ||
             (el.getAttribute('role') || '').toUpperCase() === 'LINK' ||
-            el.matches('button[class*=secondary]')
+            el.matches('button[class*=secondary]') ||
+            this.isCustomWebElementLink(el)
```

Comment:
Not a blocker: should we have a function to check for links that includes the custom web element check rather than having these inline conditionals plus the `isCustomWebElementLink`?